packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -168,6 +168,15 @@ export class Button extends FormAssociatedButton {
     private handleFormReset = () => {
         this.form?.reset();
     };
+
+    public default: HTMLSlotElement;
```

Comment:
I think the prop name should be more descriptive, "button.default" in a prop list doesn't give someone much to go on.

Reply:
I think "content" or the ref is reasonable considering it maps to the part name. Otherwise, "defaultSlot" is fairly verbose. @litteredwitherrors we need to do a quick publish but would love to get this small update in so we can unblock a few things in Fluent UI. Could we make this change today?